wg-manifests/charter.md

```diff
@@ -0,0 +1,100 @@
+# WG Manifests Charter
+
+This charter adheres to the conventions, roles and organization management
+outlined in [wg-governance].
+
+## Scope
+
+- Provide a catalog (centralized repository) of Kubeflow application manifests.
+- Provide a catalog of third-party apps for common services.
+- Provide documentation to help users install a set, or all of the included
+  apps.
+
+### In scope
+
+#### Code, Binaries and Services
+
+- Maintain a set of manifests that will allow users to install Kubeflow apps and
+  common services easily on Kubernetes, either on the cloud or on-prem, without
+  depending on external cloud services or closed source solutions. These
+  manifests will be developed with certain principles in mind, which are
+  outlined in the last bullet. Those include:
+    1. A common set of manifests for the current five (5) official Kubeflow
+       applications:
+        - Training Operators
+        - Kubeflow Pipelines (KFP)
+        - Notebooks
+        - KFServing
+        - Katib
+    1. Manifests for a set of specific common services:
+       - Istio
+       - KNative
+       - Dex
+       - Cert-Manager
+    1. Manifests for common, currently unmaintained, apps:
+        - Central Dashboard
+        - Profile Controller
+        - PodDefaults Controller
+- Maintain documentation and instructions for installing a set or all of the
+  apps. Documentation will not depend on any opinionated deployment tool. It
+  will only depend on `kustomize` and `kubectl`. This way, users and platform
+  owners can easily build on top and use their favorite opinionated deployment
+  tool (e.g., Argo, kapp, kpt, kfctl, FluxCD, etc.) with Kubeflow.
+- The maintained manifests will follow certain principles:
+    - The flow of work / separation of responsibilities will be the following:
+        1. Application owners publish manifests in their repos.
+        1. WG copies and tracks upstream manifests in the manifests repo. They
```

Comment:
I would assume an automated way to do this, at least in master branch?

Reply:
@PatrickXYS you have selected 3 lines, so I'm not sure to what you're referring 😅 
Can you please clarify, perhaps by quoting the part the question is referring to?

Reply:
Oh sorry, should be this line

```
1. WG copies and tracks upstream manifests in the manifests repo. They
form a base `kustomization`.
```

Reply:
@PatrickXYS yes, this should be automated.